apps/web/pages/getting-started.tsx

```diff
@@ -377,12 +377,12 @@ export default function Onboarding(props: inferSSRProps<typeof getServerSideProp
                   </label>
                   <p className="text-sm leading-tight text-gray-500 dark:text-white">
                     {t("current_time")}:&nbsp;
-                    <span className="text-black">{dayjs().tz(selectedTimeZone).format("LT")}</span>
+                    <span className="text-black">{dayjs().tz(dayjs.tz.guess()).format("LT")}</span>
                   </p>
                 </section>
                 <TimezoneSelect
                   id="timeZone"
-                  value={selectedTimeZone}
+                  value={dayjs.tz.guess()}
```

Comment:
Also this doesn't make sense as this change would lock the timezone select to only show the guessed timezone instead of what's been selected by the user

Reply:
Sorry about this @baileypumfleet. PR has been updated now, kindly look into it again